curl-helper.c

```diff
@@ -3995,15 +3995,19 @@ value caml_curlm_remove_finished(value v_multi)
   }
 }
 
-value caml_curl_multi_wait(value v_multi)
+value caml_curl_multi_wait(value v_timeout_ms, value v_multi)
 {
   CAMLparam1(v_multi);
   CURLM *multi_handle = CURLM_val(v_multi);
+  int timeout_ms = 1000;
   int numfds = -1;
   CURLMcode ret;
 
+  if (v_timeout_ms != Val_int(0))
+    timeout_ms = Int_val(Field(0, v_timeout_ms));
```

Comment:
%) this is a crash. Better do it on ocaml side

Reply:
Oops, indeed!